lib/course_api.rb

```diff
@@ -0,0 +1,193 @@
+require 'terms'
+require 'json'
+require 'nokogiri'
+require 'faraday'
+require 'faraday/retry'
+
+# Get course term information and then parse
+class CourseAPI
```

Comment:
Can this be moved into app/services/course_api.rb?  I'm not a fan of `lib/` as it says nothing about the role of this class.

Reply:
I can try that. I was following the general pattern of the code that already exists (e.g. course_work_courses is already in lib) so put this code there as well.

Reply:
Moved this to app/services